scss/_pagination.scss

```diff
@@ -105,6 +105,21 @@
   }
 }
 
+// Boosted mod
+.page-ellipsis {
+  display: flex;
+
+  &::after {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: calc((var(--#{$prefix}pagination-icon-size) + var(--#{$prefix}pagination-border-width) * 2)); // stylelint-disable-line function-disallowed-list
+    content: "...";
```

Comment:
We may have an a11y problem here related to WCAG 2.1 Success Criterion 1.3.1 Info and Relationships (Level A). It stated that all content must be available in text even when the CSS stylesheets are disabled.

If we disabled our stylesheets, the result is: 
![screenshot-deploy-preview-1931--boosted netlify app-2023 04 04-11_39_52](https://user-images.githubusercontent.com/92363071/229753949-e2987001-c9ee-42ad-b02a-c27078448dd2.png)

So:
- 'Previous' and 'Next' are not displayed at all (please note that it's the case for all examples, not only the current one)
- Pages 2, 3, 4, 6, 7, 8 and 9 are not accessibles too.

I recommend to double check with @Aniort.

Reply:
For the 3 dots it doesn't bring much to ear it so if I remember well it was fine to not have it vocalized or displayed.
Previous and Next are set via aria, not sure that we can change it easily. As it was anterior to this PR so let's tackle this inside another PR!
This class is used to help people building a fine design with the pages they provide. If they don't provide a smart bunch of pages, pages that aren't sent won't appear with or without CSS. But let's wait for an expert pov !

Reply:
Will be fixed by #2102.